### DISCUSSION TITLE
Remote working directory akin to vscode remote-ssh extension

### DISCUSSION THREAD

**Discussion by rgl** [gitextensions/gitextensions#9859] [Q&A] [answered]
#  Feature description

Is it possible/feasible to be able to open a remote working directory instead a local one?

Something alike the [Visual Studio Code Remote SSH extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh).

The idea is to open a remote directory in the raspbian/debian linux raspberry pi using ssh/sftp and be able to manage the git repository and working directory using Git Extensions.

**Comment by pmiossec** [comment]
No.

But perhaps you will have some luck with: https://github.com/billziss-gh/sshfs-win

**Comment by gerhardol** [comment, marked as answer]
Unfortunately no.
It could maybe be possible to wrap Git calls (in Git configuration, using OpenSSH/PuTTY), without changes in GE.
However, GE access the local environment directly too so functionality would be incomplete.

**Comment by badrshs** [comment]
I tried to use sshfs-win with  gitextension . but it didn't work . git extension wasn't able to read the git  and it said conflicts and no commit found on this repository.

**Comment by gerhardol** [comment]
In master repos in \\wsl$ work fine.
To get sshfs or something working will be more work as GE in some situations use Windows Git and access local files.
